Configuration/AlCa/python/autoCond.py

```diff
@@ -2,29 +2,27 @@
 
     ### NEW KEYS ###
     # GlobalTag for MC production with perfectly aligned and calibrated detector for Run1
-    'run1_design'                  : '113X_mcRun1_design_v3',
+    'run1_design'                  : '120X_mcRun1_design_v1',
```

Comment:
These changes in autoCOnd GTs must be blessed by @cms-sw/alca-l2 
I suspect that going back to older versions is outside the scope of this PR, and should be reverted

Reply:
AlCa is giving the blessing :) `120X_mcRun1_design_v1` is a newer version than `113X_mcRun1_design_v3`, please note the CMSSW version numbering in the beginning of the string. As explained in the PR description, the differences between the GTs can be found in the attached google sheet. Specifically this one is row 5, https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/113X_mcRun1_design_v3/120X_mcRun1_design_v1
as you can see the difference is only between the desired tags.